tests/functional/Chrome/ChromeDevToolsDriverTest.php

```diff
@@ -7,8 +7,6 @@
 
 /**
  * @group exclude-saucelabs
- * @group exclude-firefox
- * @group exclude-edge
```

Comment:
based on setUp method

Reply:
This unfortunately must be kept for any skipped tests - otherwise the tests are initialized on sauce labs but their status is never set finished (because they are lates skipped). And some unfinished tests cause saucelabs dashboard to report the whole build as failed... So we must not even star the tests.